R/render-plot.R

```diff
@@ -55,37 +55,20 @@ renderPlot <- function(expr, width='auto', height='auto', res=72, ...,
 
   args <- list(...)
 
-  if (is.function(width))
+  if (is.reactive(width))
+    widthWrapper <- width
+  else if (is.function(width))
```

Comment:
Why do we turn functions into reactives but turn constant values into functions (below)?

Reply:
We want the functions to be cached, so they go into reactives. For the constants, using a reactive rather than a function would just add overhead with no benefit.